gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,627 @@
+Metrics-server is running at https://34.30.50.119/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+----
++
+If the cluster is not up, the command can return a result similar to this one:
++
+----
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+error: the server doesn't have a resource type "services"
+----
++
+For additional information, see <<check-cluster>>.
+
+Troubleshoot errors returned by a `helm` command:
+
+[[upgrade-failed]]
+* If the Helm upgrade command fails to generate a `Service` resource of the `LoadBalancer` type, the command hangs for 5 minutes (due to the `--wait`) flag in the command and produces the following timeout error:
++
+----
+Error: UPGRADE FAILED: timed out waiting for the condition
+----
++
+This error typically means that your cluster configuration is incorrect or that your cluster configuration tool does not support this process. See <<k8-cluster-reqs>>.
+
+* If the `registration.yaml` file is invalid when you pass it to the Helm during deployment of the Flex Gateway container, the following error results:
++
+----
+Error: UPGRADE FAILED: execution error at (flex-gateway/templates/deployment.yaml:4:10):
+registerSecretName, registration.content or registration.secretName is required!
+----
++
+When this error occurs, check the `registration.yaml`. Make sure the file is not empty. If the file is empty, re-register your Flex Gateway container with a new, unique name to generate a new `registration.yaml` (see <<register-flex>>) and rerun the procedures that follow the registration step. 
+
+//TODO: this was from Israel. Not clear what upgrade error and 
+//      why we would uninstall the ingress for the namespace. 
+//Israel: "If you receive an upgrade error when execute the command, you can  
+//         remove the ingress released chart or omit the released argument:"
+//
+//## Command to remove ingress chart released
+//helm -n <namespace> uninstall ingress
+//
+// TODO: WHEN WOULD WE OMIT -i --create-namespace --wait?
+//## Command without install ingress chart arguments
+// helm -n <namespace> upgrade \
+//ingress flex-gateway/flex-gateway \
+//--set-file registration.content=<registration_file>
+
+[[check-cluster]]
+== Check Your Cluster
+
+Before deploying a Flex Gateway container to your Kubernetes cluster in <<deploy-flex>>, you must make sure that your cluster is ready. To check, you can use `kubectl` to list and select (use) a cluster and to get a list of nodes that the cluster uses. If any steps in this section fail, the deployment process is likely to produce errors.
+
+To prevent potential timeout and connectivity errors when running procedures in this guide, avoid relying on simpler tools, such as minikube, to create a local cluster unless you are experienced with the tool's cluster configuration details and have the expertise necessary to make the cluster meet the <<k8-cluster-reqs>>. 
```

Comment:
```suggestion
To prevent potential timeout and connectivity errors when running procedures in this guide, your cluster must meet the <<k8-cluster-reqs>>. 
```